clients/src/main/java/org/apache/kafka/common/metrics/KafkaMetric.java

```diff
@@ -28,7 +59,14 @@ public final class KafkaMetric implements Metric {
     private final MetricValueProvider<?> metricValueProvider;
     private MetricConfig config;
 
-    // public for testing
```

Comment:
Let's not remove this comment. It's useful to indicate why this constructor is not private.

Reply:
thanks for the comment! 
I see, let me add that back